packages/core/src/createConfig.ts

```diff
@@ -26,6 +26,9 @@ export type CreateConfigParameters = {
   websocketPort?: number
   viemClient?: PublicClient
   adminKey?: string
+  apiSecret?: string
+  apiKey?: string
```

Comment:
nit: can we name these `authServerApiKey`, `authServerApiSecret`? it's already a bit confusing to disambiguate w/ `adminKey` and this way if we add more relayer extensions w/ auth, we can add those parameters more cleanly.